### ISSUE TITLE
openvino error in loading network (with custom op)

### ISSUE THREAD

**Issue by DayDayupupupup** [triton-inference-server/server#3142]
**Description**
In my model, there are some ops that OpenVino does not support. So I looked at the documentation on how OpenVino implements custom layers：
https://docs.openvinotoolkit.org/2021.2/openvino_docs_HOWTO_Custom_Layers_Guide.html
I implemented FFT OP with reference to the above documentation, and I have verified that wnet_20.xml can infer in the OpenVino 2021.2 environment. 
But in Triton, it got an error：
```python
E0722 09:53:30.635025 529 model_repository_manager.cc:1215] failed to load 'fft' version 1: Internal: openvino error in loading network : Unsupported primitive of type: FFT name: lambda_2/IFFT2D
```
**Triton Information**
Triton version：21.06

**To Reproduce**
I rename wnet_20.xml & wnet_20.bin to model.xml & model.bin , and model repository is:
```python
fft/
├── 1
│   ├── model.bin
│   └── model.xml
└── config.pbtxt
```
config.pbtxt:
```python
name: "fft"
backend: "openvino"
input {
  name: "input_1"
  data_type: TYPE_FP32
  dims: 1
  dims: 2
  dims: 256
  dims: 256
}
output {
  name: "conv2d_44/BiasAdd/Add"
  data_type: TYPE_FP32
  dims: 1
  dims: 1
  dims: 256
  dims: 256
}
instance_group {
  count: 1
  kind: KIND_CPU
}
parameters: {
key: "CPU_EXTENSION_PATH"
value: {
string_value:"/data/libtemplate_extension.so"
}
}
```
fft op info in model.xml:
```python
<layer id="157" name="lambda_2/IFFT2D" type="FFT" version="extension">
                        <data inverse="1"/>
                        <input>
                                <port id="0">
                                        <dim>1</dim>
                                        <dim>2</dim>
                                        <dim>256</dim>
                                        <dim>256</dim>
                                </port>
                        </input>
                        <output>
                                <port id="1" precision="FP32">
                                        <dim>1</dim>
                                        <dim>2</dim>
                                        <dim>256</dim>
                                        <dim>256</dim>
                                </port>
                        </output>
                </layer>
```
I can confirm that wnet_20.xml works in the openVino 2021.2 environment, proving that libtemplate_extension.so is correct.  And from  the triton’s log， triton also loaded libtemplate_extension.so ：
```python
I0722 09:53:30.461096 529 openvino.cc:286] CPU (MKLDNN) extensions is loaded/data/libtemplate_extension.so
```
So why the FFT model can‘t be loaded？

**Comment by CoderHam** [comment]
cc @tanmayv25

**Comment by tanmayv25** [comment]
There is an open bug in OV backend for models with custom ops. See the discussion [here](https://github.com/triton-inference-server/server/issues/3134).

The OV backend is still in beta quality and does not have sufficient testing.

**Comment by tanmayv25** [comment]
@DayDayupupupup  The custom op support in OV backend was recently fixed [#27](https://github.com/triton-inference-server/openvino_backend/pull/27). Can you check whether this fixes your issue?

**Comment by tanmayv25** [comment]
Closing the issue because of inactivity. Please re-open if the issue persists.
